src/style.css

```
@@ -6768,9 +7173,12 @@ body {
   font-size: 0.94rem;
   font-weight: 600;
   color: var(--workspace-title-color);
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
+  white-space: normal;
+  overflow: visible;
+  text-overflow: clip;
+  overflow-wrap: anywhere;
```

Stylelint (17.5.0)

[error] Unexpected duplicate "overflow-wrap" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)